bot/utils/checks.py

```diff
@@ -1,6 +1,7 @@
 import datetime
 import logging
-from typing import Callable, Container, Iterable, Optional
+from collections.abc import Iterable
+from typing import Callable, Container, Optional
```

Comment:
`Container` should be imported from `collections.abc` since it's deprecated in `typing`.